### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
## Summary

Fix typos found via:
- `codespell -S package-lock.json -L inflight,ue,falsy`
- `typos --format brief`

## Checklist

- [ ] Tests have been included for all bug fixes or added functionality.
- [ ] The `changelog.rst` has been updated with any significant changes.

### CODE REVIEW

File: src/py/reactpy/tests/test_core/test_component.py [hunk 1]
```diff
@@ -56,18 +56,18 @@ def Hello():
 
 async def test_pre_tags_are_rendered_correctly(display: DisplayFixture):
     @reactpy.component
-    def PreFormated():
+    def PreFormatted():
         return reactpy.html.pre(
             {"id": "pre-form-test"},
             reactpy.html.span("this", reactpy.html.span("is"), "some"),
-            "pre-formated",
+            "pre-Formatted",
```
Comment (rmorshea) [inline src/py/reactpy/tests/test_core/test_component.py]:
```suggestion
            "pre-formatted",
```

File: src/py/reactpy/tests/test_core/test_component.py [hunk 2]
```diff
@@ -56,18 +56,18 @@ def Hello():
...
 
-    await display.show(PreFormated)
+    await display.show(PreFormatted)
 
     pre = await display.page.wait_for_selector("#pre-form-test")
 
     assert (
         await pre.evaluate("node => node.innerHTML")
-    ) == "<span>this<span>is</span>some</span>pre-formated text"
+    ) == "<span>this<span>is</span>some</span>pre-Formatted text"
```
Comment (rmorshea) [inline src/py/reactpy/tests/test_core/test_component.py]:
```suggestion
    ) == "<span>this<span>is</span>some</span>pre-formatted text"
```

File: src/py/reactpy/tests/test_core/test_events.py [hunk 3]
```diff
@@ -183,7 +183,7 @@ async def on_click(event):
     await display.page.wait_for_selector("#complete")
 
 
-async def test_can_stop_event_propogation(display: DisplayFixture):
+async def test_can_stop_event_Propagation(display: DisplayFixture):
```
Comment (rmorshea) [inline src/py/reactpy/tests/test_core/test_events.py]:
```suggestion
async def test_can_stop_event_propagation(display: DisplayFixture):
```

**Review by rmorshea — COMMENTED** [review]
Wow that worked remarkably well. Thanks!

Do you know if there's a VSCode extension that uses these same tools? I've tried [this](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) in the past but it just complains _way_ too much. Perhaps there's a way to configure it properly and I've just been too lazy to do it though.

**Review by rmorshea — APPROVED** [review]

**Comment by Archmonger** [comment]
@rmorshea I already have codespell as a plugin for mkdocs in `reactpy-django` (using `mkdocs-spellcheck`).

I will be porting the same thing to `reactpy`.

**Comment by kianmeng** [comment]
> Do you know if there's a VSCode extension that uses these same tools? I've tried [this](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) in the past but it just complains _way_ too much. Perhaps there's a way to configure it properly and I've just been too lazy to do it though.

Maybe through CI?

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**